libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/NavigationMapRoute.java

```diff
@@ -170,10 +74,14 @@ public NavigationMapRoute(@Nullable MapboxNavigation navigation, @NonNull MapVie
    * @param belowLayer optionally pass in a layer id to place the route line below
    * @param vanishRouteLineEnabled determines if the route line should vanish behind the puck during navigation.
    */
-  public NavigationMapRoute(@Nullable MapboxNavigation navigation, @NonNull MapView mapView,
-                            @NonNull MapboxMap mapboxMap, @StyleRes int styleRes,
-                            @Nullable String belowLayer, Boolean vanishRouteLineEnabled,
-                            @Nullable MapRouteLineInitializedCallback routeLineInitializedCallback) {
+  private NavigationMapRoute(@Nullable MapboxNavigation navigation,
+                             @NonNull MapView mapView,
+                             @NonNull MapboxMap mapboxMap,
+                             @NonNull LifecycleOwner lifecycleOwner,
+                             @StyleRes int styleRes,
+                             @Nullable String belowLayer,
+                             Boolean vanishRouteLineEnabled,
```

Comment:
Boolean is an object which could be null can this be changed to the primitive boolean?

Reply:
great catch, fixed!